.github/ISSUE_TEMPLATE/enhancement.yaml

```diff
@@ -0,0 +1,80 @@
+name: Feature Request
+description: Request a new code example
+title: "[Enhancement]: <DESCRIPTIVE TITLE HERE>"
+labels: ["type/enhancement"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## ⚠️ Warning
+        Issues that do not follow the below template will be marked `status/back-to-author` and assigned back to the creator.
+        
+        ---
+        
+        # ✨ Code Example Details
+  - type: textarea
+    id: context
+    attributes:
+      label: Background story
+      description: Take a moment and give us some context! Why is this example important to you?
+      placeholder: Be specific! Ctrl+Copy screenshots!
+    validations:
+      required: true
+  - type: textarea
+    id: accomplishment
+    attributes:
+      label: What does this example accomplish?
+      description: Provide a clear and brief description of what the code example should do. For example, "I want to upload a file to the cloud."
+      placeholder: Be specific! Ctrl+Copy screenshots!
+    validations:
+      required: true
+  - type: input
+    id: which-services
+    attributes:
+      label: Which AWS service(s)?
+      description: List out the [AWS service(s)](https://aws.amazon.com/products/) this code will engage.
+      placeholder: Be specific!
+    validations:
+      required: true
+  - type: checkboxes
+    id: lingo-type
+    attributes:
+      label: Which AWS SDKs or tools?
+      options:
+        - label: All languages
+        - label: .NET
+        - label: Go v2
+        - label: Java
+        - label: Java v2
+        - label: JavaScript
+        - label: JavaScript v3
+        - label: Kotlin
+        - label: PHP
+        - label: Python
+        - label: Ruby
+        - label: Rust
+        - label: Swift
+        - label: Not applicable
+  - type: textarea
+    id: which-examples
+    attributes:
+      label: Are there any existing code examples could be leveraged?
```

Comment:
label: Are there existing code examples to leverage?

Reply:
Done.